types/index.d.ts

```diff
@@ -229,6 +229,27 @@ declare namespace jest {
      */
     toBeObject(): R;
 
+    /**
```

Comment:
Sorry, I know this file is a bit painful to update. There are two interfaces that need to be updated here: `Matchers` and `Expect`. It needs `Expect` updated yet.